src/EasyList/ParseAdd.cs

```diff
@@ -1,10 +1,46 @@
 ﻿using EasyList.Enums;
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
 namespace EasyList
 {
+    public static class OptionParser
+    {
+        public static IEnumerable<(string option, string value)> ParseOptions(string[] options, string[] inputs)
```

Comment:
Would be better to use a dictionary here.
So you can do:
```c#
var options = ParseOptions(...);
var description = options["description"];
```
instead of
```c#
var description = options.FirstOrDefault(x => x.option == "description");
```

Reply:
I have actually done a dictionary version of this with some improvments.  https://github.com/chrisK00/PrettyConsoleHelper/blob/master/PrettyConsoleHelper/InputHelper.cs search for ParseOptions. Should i copy paste that solution in to here instead?